tests/e2e/test_testing.py

```diff
@@ -274,3 +274,18 @@ def test_skipping(protostar: ProtostarFixture, copy_fixture: CopyFixture):
     result = protostar(["test", "tests"])
     assert "SKIP" in result
     assert "REASON" in result
+
+
+@pytest.mark.usefixtures("init")
+def test_max_steps(protostar: ProtostarFixture, copy_fixture: CopyFixture):
```

Comment:
Can we use integration tests for this instead? E2E tests are super slow..

Reply:
Done